ideas/83-energy-efficient.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: 83-energy-efficient
 title: Energy Efficient Status
-status: Active
+status: Limbo
```

Comment:
#83 (energy efficient) is in progress and active :)

Reply:
if I'm not wrong, the point was to claim on every bi-weekly update that every Idea is still "in progress". So your new PR should revert the status of your Idea to `Active`.

Reply:
Max is right. E.g. in 83 contributors has changed quite a bit afaik, it'd be great if we can keep this live. Hopefully it is a bit easier now with only updating one single file. Can ask swarm to review PR as well so should be 5m job hopefully!

Reply:
ah, makes sense :)
so, I'm approving this PR and then I'm submitting my own, right?

Reply:
yep!